src/content/developers/tutorials/monitoring-geth-with-influxdb-and-grafana/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Monitoring Geth with InfluxDB and Grafana
+description: 
+author: "Mario Havel"
+tags: ["clients", "geth", "nodes"]
+skill: intermediate
+lang: en
+sidebar: true
+---
+
+Monitoring your node is a best practice which can help you to be informed about its performance, status, identify potential issues and even alert you if some problem occurs. Client implementations including [Geth](https://geth.ethereum.org/) offer thorough metrics collection and reporting. 
+
+## Prerequisites {#prerequisites}
+
+* Tutorial assumes that you are already running an instance of Geth client. 
+* Most of the steps and examples are for linux environment, basic terminal knowledge will be helpful. 
+* To get familiar with Geth's suite of metrics, its use for debugging and why it might be important to you, check talk [Monitoring an Ethereum infrastructure by Péter Szilágyi](https://www.youtube.com/watch?v=cOBab8IJMYI).
+
+
+## Monitoring stack {#monitoring-stack}
+
+Ethereum client collects various metrics from which a chronological database can be created. This database will be then read by data visualizing software. This means we need a stack of different tools working together. There are multiple variations of monitoring stack available with software like Prometheus (pull model), InfluxDB (push model), Telegraf, Grafana, Datadog, Chronograf, etc. 
```

Comment:
An Ethereum client collects lots of data which can be read in the form of a chronological database. To make monitoring easier, you can feed this into data visualisation software. There are multiple options available:
- Prometheus (pull model)
- InfluxDB (push model)
- Telegraf
- Grafana
- Datadog
- Chronograf

Can we make these links?